resources/DefaultConfig.ini

```diff
@@ -4,7 +4,7 @@ LoadMap=
 ; if true -> Pool allocation is not used when calling Allocate/Free, just use regular dynamic allocation. 
 ; if false -> Use pool allocation.
 DisableMemoryPool=false
-RespawnTime = 8.0
```

Comment:
Yes, the default behavior should be not to override the component in the map with the config value. Any negative value is the same as having no value at all, i.e. take the CapturePointGameMode value.